rj_gameplay/rj_gameplay/tactic/line_tactic.py

```diff
@@ -13,10 +12,14 @@ def __init__(self, world_state: stp.rc.WorldState):
 
         # compute move points
         # TODO: make start on side of the field, so this Tactic is actually useful during penalty situations
-        start = (3.0, 0.0)
+        # access the length and the width of the field (minus border), and starts robots there
+        start = (
+            (world_state.field.width_m - world_state.field.border_m) / 2,
```

Comment:
can you switch this minus to a plus so the robots stand slightly outside of the field lines?